visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -1939,6 +1959,35 @@ namespace pcl
         vtkSmartPointer<vtkRenderWindowInteractor> interactor_;
 #endif
       private:
+        /** \brief Internal function for renderer setup
+         * \param[in] vtk renderer
+         */
+        void setupRenderer (vtkSmartPointer<vtkRenderer> ren);
+
+        /** \brief Internal function for setting up FPS callback
+         * \param[in] vtk renderer
+         */
+        void setupFPSCallback (const vtkSmartPointer<vtkRenderer>& ren);
+
+        /** \brief Internal function for setting up render window
+         * \param[in] name the window name
+         */
+        void setupRenderWindow (const std::string& name);
+
+        /** \brief Internal function for setting up interactor style
+         */
+        void setupStyle ();
+
+        /** \brief Internal function for setting the default render window size and position on screen
+         */
+        void setDefaultWindowSizeAndPos ();
+
+        /** \brief Internal function for setting up camera parameters
+         * \param[in] argc
+         * \param[in] argv
+         */
+        void setupCamera (int &argc, char **argv);
```

Comment:
Small comment: I know this PR is only refactoring, but this signature is completely cryptic for me. Do you have any idea which format one passes to these camera parameters? Is there any way we can add some documentation to improve it this an the other signatures which pass these? From our current docs there's nothing on it.

Reply:
As far as I understand, we pass the program arguments, which may contain -cam "<name>.cam". It is good idea to add the documentation for it. I will do it, but in the another merge request. We are using my change in our project at the moment, and it will be nice to use pcl master repo instead of my fork.